ui/src/components/CopyToClipboard/CopyToClipboard.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { Button } from 'react-bootstrap';
+import CopyToClipboard from 'react-copy-to-clipboard';
```

Comment:
The file is called `CopyToClipboard` but the function is called `CopyToClipb` is that by purpose it would somewhat more sense if it was called `CopyToClipboard`

Reply:
yes CopyToClipboard is also the default import  name of the react library
Maybe i can do import As

Reply:
I do import as, as require one file change